src/main/groovy/com/marklogic/gradle/task/tests/RoxyTest.groovy

```diff
@@ -0,0 +1,53 @@
+package com.marklogic.gradle.task.tests
+
+import com.marklogic.appdeployer.AppConfig
+import com.marklogic.gradle.task.MarkLogicTask
+import com.marklogic.mgmt.ManageClient
+import com.marklogic.mgmt.ManageConfig
+import com.marklogic.rest.util.Fragment
+import org.gradle.api.tasks.TaskAction
+
+
+class RoxyTest extends MarkLogicTask{
+
+	@TaskAction
+	void runAllTests() {
+		AppConfig config = getAppConfig()
+		String testResultDir = getProject().getProperties().get("mlTestResultDir")
```

Comment:
Use "mlRoxyTestResultPath" . I'm trying to standardize on "Path" instead of "Dir", along with prefixing any Roxy-specific property with "mlRoxy".

Reply:
Also, it seems this could have a default value, like "build/roxy-tests". Then a user doesn't need to specify a property unless they want to override the path. I think the property should specify the full path too, such that the task doesn't have to append anything to it.